packages/reports/addon/controllers/reports/report.js

```diff
@@ -19,6 +19,11 @@ export default Controller.extend({
    */
   showSaveAs: false,
 
+  /**
+   * @property {Boolean} isFiltersCollapsed
+   */
+  isFiltersCollapsed: false,
```

Comment:
Any reason why this needs to be at the route level instead of in the `report-builder` component?

Reply:
it's passed down to `report-view` as well to trigger the chart resize when changing the collapsed state